### DISCUSSION TITLE
Unable to share pictures under photoprism

### DISCUSSION THREAD

**Discussion by hmouras** [photoprism/photoprism#3988] [Q&A]
Dear Community, 
I have photoprism installed on an OpenMediaVault 6 NAS system. Everything is working well, but I am unable to share a picture through a sreated shared link. I can create a link but when I do enter this link in a browser, this does not show any picture. 
Any idea ? 
Thank you so much, 
Best regards, 
Harold

**Comment by graciousgrey** [comment]
Hello :wave:,

This might be related to your configuration. How does the url of the sharing link look like? And what is your configured site url? Do you use a reverse proxy?

**Comment by hmouras** [comment]
Thank you for your message. 
First, I can only create link to share albums, but not only one picture for example, right ? 
Second, the link created by photoprism is like http://photoprism.hmpouras.duckdns.org/s/rfmkldjww2/juillet-2023 whereas my good adress is http://photoprism.hmouras.duckdns.org. Where is it possible to set up this complementary letter added in the ling to my domain ? 
Thanks, 
Best regards, 
Harold

**Reply by graciousgrey** [reply]
Yes, at this time sharing links can only be created for albums. The possibility to share single files via links is planned.
Is it possible that you have a typo in the site url configuration?

You can configure it with the [PHOTOPRISM_SITE_URL](https://docs.photoprism.app/getting-started/config-options/#site-information) variable.

The sharing link contains "hmpouras" whereas your other url contains "hmouras".
